src/legacy/networks.ts

```diff
@@ -48,6 +49,8 @@ const HASKELL_SHELLEY = {
   IS_MAINNET: true,
 
   EXPLORER_URL_FOR_ADDRESS: (address: string) => `https://cardanoscan.io/address/${address}`,
+  EXPLORER_URL_FOR_TOKEN: (fingerprint: string) =>
+    fingerprint.length > 0 ? `https://cardanoscan.io/token/${fingerprint}` : `https://cardanoscan.io/tokens`,
```

Comment:
`''` = primary token, it will fallback to `/tokens` (page that list all tokens) taking suggestions

Reply:
should `EXPLORER_URL_FOR_TOKEN`, `EXPLORER_URL_FOR_ADDRESS`, etc be lowercase?

Reply:
camelCase specifically?

Reply:
because it is a function i mean

Reply:
yap, this will be refactored while moving the network -> wallet, there is a spec for that